doc/src/manual/profile.md

```diff
@@ -61,80 +61,84 @@ available, but here we'll use the text-based display that comes with the standar
 
 ```julia
 julia> Profile.print()
-      23 client.jl; _start; line: 373
-        23 client.jl; run_repl; line: 166
-           23 client.jl; eval_user_input; line: 91
-              23 profile.jl; anonymous; line: 14
-                 8  none; myfunc; line: 2
-                  8 dSFMT.jl; dsfmt_gv_fill_array_close_open!; line: 128
-                 15 none; myfunc; line: 3
-                  2  reduce.jl; max; line: 35
-                  2  reduce.jl; max; line: 36
-                  11 reduce.jl; max; line: 37
+80 ./event.jl:73; (::Base.REPL.##1#2{Base.REPL.REPLBackend})()
+ 80 ./REPL.jl:97; macro expansion
+  80 ./REPL.jl:66; eval_user_input(::Any, ::Base.REPL.REPLBackend)
+   80 ./boot.jl:235; eval(::Module, ::Any)
+    80 ./<missing>:?; anonymous
+     80 ./profile.jl:23; macro expansion
+      52 ./REPL[1]:2; myfunc()
+       38 ./random.jl:431; rand!(::MersenneTwister, ::Array{Float64,3}, ::Int64, ::Type{B...
+        38 ./dSFMT.jl:84; dsfmt_fill_array_close_open!(::Base.dSFMT.DSFMT_state, ::Ptr{F...
+       14 ./random.jl:278; rand
+        14 ./random.jl:277; rand
+         14 ./random.jl:366; rand
+          14 ./random.jl:369; rand
+      28 ./REPL[1]:3; myfunc()
+       28 ./reduce.jl:270; _mapreduce(::Base.#identity, ::Base.#scalarmax, ::IndexLinear,...
+        3  ./reduce.jl:426; mapreduce_impl(::Base.#identity, ::Base.#scalarmax, ::Array{F...
+        25 ./reduce.jl:428; mapreduce_impl(::Base.#identity, ::Base.#scalarmax, ::Array{F...
 ```
 
 Each line of this display represents a particular spot (line number) in the code. Indentation
 is used to indicate the nested sequence of function calls, with more-indented lines being deeper
 in the sequence of calls. In each line, the first "field" indicates the number of backtraces
 (samples) taken *at this line or in any functions executed by this line*. The second field is
-the file name, followed by a semicolon; the third is the function name followed by a semicolon,
-and the fourth is the line number. Note that the specific line numbers may change as Julia's
+the file name, followed by a colon and the line number; the third is the function name.
+Note that the specific line numbers may change as Julia's
 code changes; if you want to follow along, it's best to run this example yourself.
 
-In this example, we can see that the top level is `client.jl`'s `_start` function. This is the
-first Julia function that gets called when you launch Julia. If you examine line 373 of `client.jl`,
-you'll see that (at the time of this writing) it calls `run_repl()`, mentioned on the second line.
-This in turn calls `eval_user_input()`. These are the functions in `client.jl` that interpret
+In this example, we can see that the top level function called is in the file `event.jl`. This is the
+function that schedules the task that runs the REPL. If you examine line 97 of `REPL.jl`,
+you'll see this calls `eval_user_input()`. This is the function that interprets
```

Comment:
Perhaps "you'll see this function calls" for clarity? Likewise perhaps "`eval_user_input()` interprets what you type..."?